cmake/find/Findodb-compiler.cmake

```diff
@@ -0,0 +1,366 @@
+  endif()
+
+  if(PARAM_SOURCE_EPILOGUE)
+    list(APPEND ODB_ARGS --cxx-epilogue "${PARAM_SOURCE_EPILOGUE}")
+  endif()
+
+  #AWP: prolog and epilogue from existing files
+  if(PARAM_ODB_PROLOGUE-FILE)
+    list(APPEND ODB_ARGS --odb-prologue-file "${PARAM_ODB_PROLOGUE-FILE}")
+  endif()
+
+  if(PARAM_HEADER_PROLOGUE-FILE)
+    list(APPEND ODB_ARGS --hxx-prologue-file "${PARAM_HEADER_PROLOGUE-FILE}")
+  endif()
+
+  if(PARAM_INLINE_PROLOGUE-FILE)
+    list(APPEND ODB_ARGS --ixx-prologue-file "${PARAM_INLINE_PROLOGUE-FILE}")
+  endif()
+
+  if(PARAM_SOURCE_PROLOGUE-FILE)
+    list(APPEND ODB_ARGS --cxx-prologue-file "${PARAM_SOURCE_PROLOGUE-FILE}")
+  endif()
+
+  if(PARAM_ODB_EPILOGUE-FILE)
+    list(APPEND ODB_ARGS --odb-epilogue-file "${PARAM_ODB_EPILOGUE-FILE}")
+  endif()
+
+  if(PARAM_HEADER_EPILOGUE-FILE)
+    list(APPEND ODB_ARGS --hxx-epilogue-file "${PARAM_HEADER_EPILOGUE-FILE}")
+  endif()
+
+  if(PARAM_INLINE_EPILOGUE-FILE)
+    list(APPEND ODB_ARGS --ixx-epilogue-file "${PARAM_INLINE_EPILOGUE-FILE}")
+  endif()
+
+  if(PARAM_SOURCE_EPILOGUE-FILE)
+    list(APPEND ODB_ARGS --cxx-epilogue-file "${PARAM_SOURCE_EPILOGUE-FILE}")
+  endif()
+
+  if(PARAM_PROFILE)
+    list(APPEND ODB_ARGS --profile "${PARAM_PROFILE}")
+  endif()
+
+  list(APPEND ODB_ARGS --output-dir "${ODB_COMPILE_OUTPUT_DIR}")
+  list(APPEND ODB_ARGS --hxx-suffix "${ODB_COMPILE_HEADER_SUFFIX}")
+  list(APPEND ODB_ARGS --ixx-suffix "${ODB_COMPILE_INLINE_SUFFIX}")
+  list(APPEND ODB_ARGS --cxx-suffix "${ODB_COMPILE_SOURCE_SUFFIX}")
+
+  if(PARAM_MULTI_DATABASE AND NOT "${ODB_COMPILE_FILE_SUFFIX}" MATCHES ".+:.+")
+    set(osuffix "${ODB_COMPILE_FILE_SUFFIX}")
+    set(ODB_COMPILE_FILE_SUFFIX)
+    foreach(db ${PARAM_DB})
+      if("${db}" MATCHES "common")
+        list(APPEND ODB_COMPILE_FILE_SUFFIX "${db}:${osuffix}")
+      else()
+        list(APPEND ODB_COMPILE_FILE_SUFFIX "${db}:${osuffix}_${db}")
+      endif()
+    endforeach()
+  endif()
+
+  foreach(sfx ${ODB_COMPILE_FILE_SUFFIX})
+    list(APPEND ODB_ARGS --odb-file-suffix "${sfx}")
+  endforeach()
+
+  foreach(dir ${PARAM_INCLUDE} ${ODB_INCLUDE_DIRS})
+    list(APPEND ODB_ARGS "-I${dir}")
+  endforeach()
+
+  file(REMOVE_RECURSE "${ODB_COMPILE_OUTPUT_DIR}")
+  file(MAKE_DIRECTORY "${ODB_COMPILE_OUTPUT_DIR}")
+
+  #AWP: TODO: add contents of *-file param as dependencies of custom_command
+  set(generated_source_files)
+  foreach(input ${PARAM_FILES})
+    get_filename_component(fname "${input}" NAME_WE)
+    set(outputs)
+
+    foreach(sfx ${ODB_COMPILE_FILE_SUFFIX})
+      string(REGEX REPLACE ":.*$" "" pfx "${sfx}")
+      string(REGEX REPLACE "^.*:" "" sfx "${sfx}")
+
+      if(NOT "${PARAM_MULTI_DATABASE}" MATCHES "static" OR NOT "${pfx}" MATCHES "common")
+        set(output "${ODB_COMPILE_OUTPUT_DIR}/${fname}${sfx}${ODB_COMPILE_SOURCE_SUFFIX}")
+        list(APPEND outputs "${output}")
+        list(APPEND generated_source_files "${output}")
+      endif()
+    endforeach()
+
+    if(HUNTER_STATUS_DEBUG)
```

Comment:
You can keep code with creating the list and just replace `message(STATUS ...)` with `hunter_status_debug`